changelog.yaml

```diff
@@ -21,6 +21,11 @@
 - version: "NEXT"
   date: TBD
   changes:
+  - title: Upgrade Go dependencies
```

Comment:
pull request number is mandatory. (jira missing if there is one).
Also I would propose to keep all fields, but leave them empty if not applicable,